tutorials/fundamentals/installation.md

```diff
@@ -58,15 +75,15 @@ conda-forge channel. You can install it with:
 conda install -c conda-forge napari
 ```
 
-### install from the master branch on Github
+### Install from the master branch on Github
 
 To install the "next-release" version from github via pip, call
 
 ```sh
 pip install "git+https://github.com/napari/napari.git#egg=napari[all]"
 ```
 
-### clone the repository locally and install in editable mode
+### Clone the repository locally and install in editable mode
```

Comment:
I don't think this content should live here - editable pip installations are for the developer docs (and these instructions are also slightly different to what's written there https://napari.org/developers/contributing.html)